### ISSUE TITLE
No accessible name on search submit button

### ISSUE THREAD

**Issue by joedolson** [WordPress/wporg-mu-plugins#163]
The search button SVG is an unnamed element, so the button has no accessible name. The button can be named using aria-label, .screen-reader-text, or via an accessible name on the SVG, but needs some kind of accessible text!

**Comment by ryelle** [comment]
This is a bug inherited from Gutenberg https://github.com/WordPress/gutenberg/pull/38136, but the fix was merged today so it should be fixed when GB 12.7 is released. We could try adding a workaround in the meantime though.

**Comment by joedolson** [comment]
Seems like waiting on 12.7 is a reasonable solution.

**Comment by ryelle** [comment]
Tagging as blocked, will re-test and close this once Gutenberg 12.7 is released.

**Comment by ryelle** [comment]
Now that 12.7 is out, this button has a label 👍🏻
